### ISSUE TITLE
请问funcs/decoder.py中154和155行中@用法有问题吗？

### ISSUE THREAD

**Issue by UCC-team** [iamlxb3/s2s_tutorial#1]
请问funcs/decoder.py中154和155行中@用法有问题吗？
            output_weights = self.embedding_project @ torch.transpose(self.embedding.weight.detach(), 1, 0)
            output = concat_output @ output_weights
会提示报错

**Comment by UCC-team** [comment]
![image](https://user-images.githubusercontent.com/35629974/50131444-15d70b80-02bd-11e9-9266-e46d290559fb.png)
